src/twisted/scripts/_twistd_unix.py

```diff
@@ -68,7 +69,7 @@ class ServerOptions(app.ServerOptions):
     def opt_version(self):
         """Print version information and exit.
         """
-        print('twistd (the Twisted daemon) %s' % copyright.version)
+        print('twistd (the Twisted daemon) {}'.format(copyright.version))
```

Comment:
Normally I would update this in a separate PR, but this change is relatively small and doesn't break anything.  you can look at this entire PR, commit by commit at https://github.com/twisted/twisted/pull/869/commits and see that I isolated the format changes.